MatrixSDK/MXRestClient.m

```diff
@@ -3673,6 +3673,32 @@ - (MXHTTPOperation*) uploadContent:(NSData *)data
                                  }];
 }
 
+- (MXHTTPOperation*) maxUploadSize:(void (^)(NSInteger maxUploadSize))success
+                           failure:(void (^)(NSError *error))failure
+{
+    MXWeakify(self);
+    return [httpClient requestWithMethod:@"GET"
+                                    path:[NSString stringWithFormat:@"%@/config", contentPathPrefix]
+                              parameters:nil
+                                 success:^(NSDictionary *JSONResponse) {
+                                     MXStrongifyAndReturnIfNil(self);
+
+                                     if (success)
+                                     {
+                                         __block NSInteger maxUploadSize;
```

Comment:
Local variables must be initialised in objc.
They may be 0 in debug but not in release build...

```suggestion
                                         __block NSInteger maxUploadSize = 0;
```


Note that ivars are initialised to 0 or nil or NO because the instance memory allocator does it.